presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -434,6 +439,10 @@ Convenience Extraction Functions
 
     This is an alias for :func:`day_of_week`.
 
+..  function:: dow(x, y) -> bigint
+
+        This is an alias for :func:`day_of_week`.
```

Comment:
which link are we referring here?

Reply:
I was referring to ``:func:day_of_week`` part appearing as a link in the documentation (e.g. https://prestosql.io/docs/current/functions/datetime.html#day_of_week) . So was wondering if it's possible to point it to a specific signature to indicate what the alias corresponds to. But I think it's reasonably intuitive because of the number of parameters.

Reply:
sphinx does not support that

Reply:
@phd3 is it a problem if it points to just function without mentioning the signature, like in current docs(it points to day_of_week())?

Reply:
yeah, it does not look like a major issue, should be fine.